lib/ccsds123b/src/constants.c

```diff
@@ -0,0 +1,21 @@
+#include <finch/ccsds123b/constants.h>
+
+int const D = 7;
+
+int const Smin;
+int const Smax = (1 << D) - 1;
+int const Smid = 1 << (D - 1);
+
+int const P = 3;
+
+int const Omega = 10;
+
+int const R = 32;
+
+int const THETA = 4;
+int const PHI;
+int const PSI;
```

Comment:
Are these variables not initialized, even though they are declared as const?? I'm confused. Maybe I missed something.

Reply:
The C standard initializes them to 0, and the compliance check prefers that variables are not explicitly initialized to 0. I think it's because that explicitly assigning 0 to a variable adds redundant operations.

Reply:
I want those variables to have the value 0 for now. When it's done I'll add documentation on why I chose the values but for now none of the values have been finalized